### DISCUSSION TITLE
Building Lean Copilot on a GPU enabled machine

### DISCUSSION THREAD

**Discussion by vanessalama09** [lean-dojo/LeanCopilot#105] [Q&A] [answered]
I'm trying to build Lean Copilot on NVIDIA cuda-compatible GPU. I was able to build locally, but on the GPU, it's throwing an error. A little about the machine:
- linux x86_64
- NVIDIA A100 GPUs

The build error:

```
ℹ [3/1126] Replayed LeanCopilot/libopenblas
info: Cloning OpenBLAS from https://github.com/OpenMathLib/OpenBLAS
info: Building OpenBLAS with `make NO_LAPACK=1 NO_FORTRAN=1 -j32`
✖ [4/1126] Building LeanCopilot/libctranslate2
info: Cloning CTranslate2 from https://github.com/OpenNMT/CTranslate2
error: no such file or directory (error code: 2)
  file: ././.lake/packages/LeanCopilot/.lake/build/lib/libctranslate2.so
Some builds logged failures:
- LeanCopilot/libctranslate2
error: build failed
```

A little help would be much appreciated!

**Comment by Peiyang-Song** [comment, marked as answer]
I just tried the latest version of Lean Copilot (v1.5.3) on a x86_64 Linux machine with CUDA-enabled GPUs, and no errors occurred during building. Could you try the latest version and see if the build works?
